Makefile

```diff
@@ -82,5 +82,9 @@ lint:
 
 fmt:
 	gofmt -s -l -w $(SRCS)
+
+clean:
+	sudo rm -rf dist static site autogen/gen.go traefik
```

Comment:
Strange, I never had a problem with deleting files created by the build container. Not in front of my laptop right now, but is this maybe something happening on some platforms other than Mac OS X only?

I'm okay with creating a clean target (though I personally never needed it). Does our Makefile call `go build` without specifying an output file somewhere below `dist/` though? If not, then I still believe `traefik` (in the root directory) should not be removed.